docs/logs/reports-and-components.md

```diff
@@ -0,0 +1,75 @@
+title: Sematext Logs Reports and Components
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
+
+Each Logs App has one default Report called **Explore**. You can create as many reports as you want with custom Components.
+
+## Logs Search and Report Menu
+
+Events, Alerts, and Logs are integrated into a single reports pane, and used to search logs, create alerts for anomalies, view events, and much more. 
```

Comment:
Hm, which events?  There are no events other than log events in the default Explore report.

Reply:
Edited.